apis/stash/v1beta1/annotations.go

```diff
@@ -1,10 +1,23 @@
 package v1beta1
 
 const (
-	BackupKey                = "backup.appscode.com"
-	LastAppliedConfiguration = BackupKey + "/last-applied-configuration"
-	VersionTag               = BackupKey + "/tag"
+	VersionTag = BackupKey + "/tag"
 	// ResourceVersion will be used to trigger restarts for ReplicaSet and RC pods
-	ResourceHash = BackupKey + "/resource-hash"
-	StashKey     = "stash.appscode.com"
+	StashKey                       = "stash.appscode.com"
+	BackupKey                      = "backup.appscode.com"
```

Comment:
All keys should use the one single prefix `stash.appscode.com`

Reply:
fixed in https://github.com/appscode/stash/pull/701